transport/src/main/java/io/netty/channel/CombinedChannelDuplexHandler.java

```diff
@@ -25,10 +25,11 @@
 import java.net.SocketAddress;
 
 /**
- *  Combines a {@link ChannelInboundHandler} and a {@link ChannelOutboundHandler} into one {@link ChannelHandler}.
+ *  Combines a {@link ChannelHandler} (which only implements inbound handling) and a {@link ChannelHandler}
+ *  (which only implements outbound handling) into one {@link ChannelHandler}.
```

Comment:
Could consider rewording to something like "Combines the inbound handling of one ChannelHandler with the outbound handling of another", just because technically either of them could be two-way already.